cloud/amazon/ec2_placement.py

```diff
@@ -0,0 +1,166 @@
+        Use list to show all or search specific placement group.
+    required: false
+    default: present
+    choices: ['present', 'absent', 'list']
+  region:
+    description:
+      - region in which the resource exists.
+    required: false
+    default: null
+    aliases: ['aws_region', 'ec2_region']
+
+author: "Slawomir Skowron (@szibis)"
+extends_documentation_fragment: aws
+'''
+
+EXAMPLES = '''
+# Basic example of adding placement group
+tasks:
+- name: add placement group
+  ec2_placement:
+      name: "example_group"
+      state: "present"
+      region: "us-east-1"
+
+# Basic example for delete placement group
+tasks:
+- name: remove placement group
+  ec2_placement:
+      name: "example_group"
+      state: "absent"
+      region: "us-east-1"
+
+# Basic example for listing all placement groups
+tasks:
+- name: list all placement group
+  ec2_placement:
+      state: "list"
+      region: "us-east-1"
+
+# Basic example for listing specific placement groups
+tasks:
+- name: list placement group
+  ec2_placement:
+      name: "example_group"
+      state: "list"
+      region: "us-east-1"
+'''
+
+import sys
+
+try:
+    import boto.ec2
+    HAS_BOTO = True
+except ImportError:
+    HAS_BOTO = False
+
+
+def main():
+    argument_spec = ec2_argument_spec()
+    argument_spec.update(dict(
+                         name=dict(required=False),
+                         strategy=dict(default='cluster',
+                                       choices=['cluster']
+                                       ),
+                         state=dict(default='present',
+                                    choices=['present', 'absent', 'list']
+                                    )
+                            )
+                        )
+    module = AnsibleModule(argument_spec=argument_spec)
+
+    if not HAS_BOTO:
+        module.fail_json(msg='boto required for this module')
+
+    name = module.params.get('name')
+    strategy = module.params.get('strategy')
+    state = module.params.get('state')
+
+    if not isinstance(name, basestring) and name:
+        module.fail_json(msg="name argument must be string")
+
+    ec2 = ec2_connect(module)
+
+    filters = {'group-name': name}
+    if name:
+        getplacements = ec2.get_all_placement_groups(filters=filters)
+    else:
+        getplacements = ec2.get_all_placement_groups()
+
+    placelist = []
+
+    for placement in getplacements:
+        placelist.append(placement.name)
+
+    if state == 'present':
+        if not name:
+            module.fail_json(msg="name argument is required when state is \
+                             present")
+        if name in placelist:
+            module.exit_json(msg="Placement group already exists with name \
+                             %s." % name, changed=False)
+        else:
+            placer = ec2.create_placement_group(name)
+        module.exit_json(msg="Placement group %s created." % name,
```

Comment:
The exit should return something useful. In this case, that might be name and strategy. 

Whatever the results of `state: list` return, the module should return similar results when creating a placement group.